### PR TITLE
#245 Fixed broken markup of created meetings

### PR DESCRIPTION
**Pull request by EugeneValko**
Screenshot_11

### CODE REVIEW

**Review by va-dym-ko-val — APPROVED** [review]

**Review by tatianahutii — APPROVED** [review]

File: frontend/src/app/modules/bookings/bookings-page/bookings-page.component.sass [hunk 1]
```diff
@@ -25,6 +25,8 @@ p, span
     color: $pale-sky
 
 .item
+    min-width: 300px
+    width: 300px
```
Comment (tatianahutii) [inline frontend/src/app/modules/bookings/bookings-page/bookings-page.component.sass]: why in px?

**Review by tatianahutii — CHANGES_REQUESTED** [review]

File: frontend/src/app/modules/bookings/bookings-page/bookings-page.component.sass (hunk 1 above)
Comment (EvgenValko) [reply frontend/src/app/modules/bookings/bookings-page/bookings-page.component.sass:29]: I think this component needs to be refactored, but it is a little hot fix before the demo, and will be rewritten in the redesign

Comment (EugeneValko) [reply frontend/src/app/modules/bookings/bookings-page/bookings-page.component.sass]: Fixed

Comment (tatianahutii) [reply frontend/src/app/modules/bookings/bookings-page/bookings-page.component.sass]: it would be better to use %

**Review by tatianahutii — APPROVED** [review]

**Review by apofis96 — APPROVED** [review]

**Pull request merged**